manifest.yml

```diff
@@ -14,4 +14,5 @@ applications:
     PERF_TEST_END_NUMBER_OF_USERS: 50
     PERF_TEST_SOAK_TEST_DURATION_MINUTES: 1
     THRESHOLD_95TH_PERCENTILE_MILLIS: 1000
-    THRESHOLD_MEAN_MILLIS: 500
\ No newline at end of file
+    THRESHOLD_MEAN_MILLIS: 500
+    SESSION_DETAILS_BASE_URL: ((session_details_url))
```

Comment:
This will only have an effect when running `cf push` from your local machine, as this manifest is not used by the CD pipeline (which writes its own).